ios/StripeTerminalReactNative.swift

```diff
@@ -513,6 +525,18 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         sendEvent(withName: ReactNativeConstants.CHANGE_CONNECTION_STATUS.rawValue, body: ["result": result])
     }
 
+    func terminal(_ terminal: Terminal, didStartReaderReconnect cancelable: Cancelable) {
+        sendEvent(withName: ReactNativeConstants.START_READER_RECONNECT.rawValue, body: nil)
+    }
+
+    func terminalDidSucceedReaderReconnect(_ terminal: Terminal) {
+        sendEvent(withName: ReactNativeConstants.TERMINAL_SUCCEED_READER_RECONNECT.rawValue, body: nil)
+    }
+
+    func terminalDidFailReaderReconnect(_ terminal: Terminal) {
+        sendEvent(withName: ReactNativeConstants.TERMINAL_FAIL_READER_RECONNECT.rawValue, body: nil)
+    }
```

Comment:
same as Android comment above, do we want to send an error here like https://github.com/stripe/stripe-terminal-react-native/blob/1870117c/ios/StripeTerminalReactNative.swift#L331-L334

cc @bric-stripe